starter/script.js

```diff
@@ -1 +1,33 @@
 // put code here!
+var tantrumFree = function() {
+  console.log("**************************")
+}
+console.log ("You've overslept one morning and realized you have 25 minutes to get yourself and your kindergartener out the door")
+var wakeUpInput = prompt("You can either jolt out of bed or push the snooze button") (Please enter 'j' or 's')
+
+if (wakeUpInput) == "j"{
+  console.log("You jolt out of bed and begin dressing your kindergartener, when she decides she doesn't want to wear your option but rather her polka-dot pink Doc McStuffin shirt with her green striped duck pants."
+}
+var (reponseToToddler) = prompt("You realize that this very moment could make or break her blossoming kindergarten reputation" and decide to \n('a') - argue with her. We're talking reputation here people!...hers and yours!\n('g') - give in, toddlers have no reputation.  This may actually increase her self-confidence, while the other parents question your parenting skills\n('n')- negotiate with her.  She may know the latest trends that you may not be hip to and needs just a little guidance.")
+
+if(responseToToddler == 'a'){
+  console.log("You argue tooth and nail with her, when you realize you've wasted time, increased your blood pressure and will inevitably be late")
+  buffer()
```

Comment:
I don't see a `buffer` function defined anywhere in your code... was that something you made and then deleted?